--- mpdscribble-0.18.1/Makefile.am.orig	2009-08-30 07:58:48.000000000 +0200
+++ mpdscribble-0.18.1/Makefile.am	2009-09-20 21:01:55.000000000 +0200
@@ -47,7 +47,7 @@
 	$(libgcrypt_LIBS) \
 	$(libglib_LIBS)
 
-AM_CPPFLAGS += -DFILE_CONF='"$(sysconfdir)/mpdscribble.conf"'
+AM_CPPFLAGS += -DFILE_CONF='"$(sysconfdir)/mpdscribble/mpdscribble.conf"'
 
 SPARSE = sparse
 SPARSE_FLAGS =
@@ -63,15 +63,15 @@
 INSTALL_SECURE = $(INSTALL) -m 600
 
 install-dist_sysconfDATA:
-	test -z "$(sysconfdir)" || $(mkdir_p) "$(DESTDIR)$(sysconfdir)"
+	test -z "$(sysconfdir)/mpdscribble" || $(mkdir_p) "$(DESTDIR)$(sysconfdir)/mpdscribble"
 	@list='$(dist_sysconf_DATA)'; \
 	for p in $$list; do \
-	    dest=$(am__strip_dir) \
-	    if test -f $(DESTDIR)$(sysconfdir)/$$dest; then \
-	        echo "$@ will not overwrite existing $(DESTDIR)$(sysconfdir)/$$dest"; \
+	    $(am__strip_dir) \
+	    if test -f $(DESTDIR)$(sysconfdir)/mpdscribble/$$f; then \
+	        echo "$@ will not overwrite existing $(DESTDIR)$(sysconfdir)/mpdscribble/$$f"; \
 	    else \
 	        if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	        echo " $(INSTALL_SECURE) $$d$$p $(DESTDIR)$(sysconfdir)/$$dest"; \
-	        $(INSTALL_SECURE) $$d$$p $(DESTDIR)$(sysconfdir)/$$dest; \
+	        echo " $(INSTALL_SECURE) $$d$$p $(DESTDIR)$(sysconfdir)/mpdscribble/$$f"; \
+	        $(INSTALL_SECURE) $$d$$p $(DESTDIR)$(sysconfdir)/mpdscribble/$$f; \
 	    fi; \
 	done
--- mpdscribble-0.18.1/doc/mpdscribble.conf.orig	2009-08-30 07:58:48.000000000 +0200
+++ mpdscribble-0.18.1/doc/mpdscribble.conf	2009-09-20 21:50:25.000000000 +0200
@@ -14,7 +14,8 @@
 # "syslog" makes mpdscribble use the local syslog daemon.  On most
 # systems, log messages will appear in /var/log/daemon.log then.
 # "-" means log to stderr (the current terminal).
-log = syslog
+#log = syslog
+log = /var/log/mpdscribble/mpdscribble.log
 
 # How verbose mpdscribble's logging should be.  Default is 1.
 verbose = 1
