--- mpdscribble-0.2.12/file.h	2008-09-03 19:52:41.000000000 +0000
+++ mpdscribble-0.2.12/file.h.	2008-09-03 19:53:00.000000000 +0000
@@ -29,9 +29,9 @@
   FILE_USR_* will be used instead if FILE_USR_CONF exists.
 */
 
-#define FILE_CONF       "/etc/mpdscribble.conf"
+#define FILE_CONF       "/etc/mpdscribble/mpdscribble.conf"
 #define FILE_CACHE      "/var/cache/mpdscribble/mpdscribble.cache"
-#define FILE_LOG        "/var/log/mpdscribble/mpdscribble.log"
+#define FILE_LOG        "/var/log/mpdscribble.log"
 #define FILE_HOME_CONF  "~/.mpdscribble/mpdscribble.conf"
 #define FILE_HOME_CACHE "~/.mpdscribble/mpdscribble.cache"
 #define FILE_HOME_LOG   "~/.mpdscribble/mpdscribble.log"
